airflow-core.yaml

```diff
@@ -0,0 +1,193 @@
+package:
+  name: airflow-core
+  version: 2.10.4
+  epoch: 0
+  description: Minimal Airflow core - Platform to programmatically author, schedule, and monitor workflows
+  dependencies:
+    runtime:
+      - bash
+      - busybox
+      - coreutils
+      - curl
+      - cyrus-sasl
+      - cyrus-sasl-dev
+      - findutils
+      - glibc-locale-en
+      - glibc-locales
+      - gnupg-utils
+      - gzip
+      - mariadb-dev
+      - openssl
+      - postgresql-client
+      - postgresql-dev
```

Comment:
Looking great, but is all of PostgreSQL dev needed?

Reply:
I see this in upstream, that confused me
```bash
(airflow)pwd
/lib/postgresql/17/lib
(airflow)ls
pgxs
(airflow)
```

Reply:
And I found this reference https://stackoverflow.com/a/7379005

Reply:
Are we sure we need postgresql-dev at buildtime? does the package build without this dependency if we drop it? Had a look a the stackoverflow article and reads more like a runtime issue.

Reply:
Actually, that's is a very good catch and I am sorry for missing it myself, we don't need most build time dep because of pip install method here that we follow, I just rebuilt without them and it worked fine, other than few python deps, it needed only busybox. Thanks again!